src/Raven.Studio/wwwroot/App/views/resources/createDatabase.html

```diff
@@ -477,6 +477,14 @@ <h3>Backup Source Configuration</h3>
                        </div>
                    </div>
                </div>
+               <div class="row margin-bottom" data-bind="collapse: useCustomS3Host">
```

Comment:
can you verify that it should be collapsed when using custom s3 host?

Reply:
We only need this when using a custom URL.
When using Amazon AWS, the default value will always work.

Reply:
ok thanks for confirming that.